src/rules/noShadowedVariableRule.ts

```diff
@@ -35,6 +35,26 @@ export class Rule extends Lint.Rules.AbstractRule {
             and \`"typeParameter"\`. Just set the value to \`false\` for the check you want to disable.
             All checks default to \`true\`, i.e. are enabled by default.
             Note that you cannot disable variables and parameters.
+
+            The option \`"temporalDeadZone"\` defaults to \`false\`. When set to \`true\` parameters, classes, enums and variables declared
```

Comment:
I think it should default to `true` to align with the other options -- they are all `true` by default, which indicates that a certain type of check is _enabled_. Here, we intend to enable checking within temporal dead zones by default; requiring the option to be `false` would make sense to opt-out of the check -- this would make the rule less strict (similar to setting `false` on the other options).

Reply:
So you basically want me to swap true and false here and in all conditions? Sounds reasonable

Reply:
Yes I would like you to swap true and false in all the new code you've added (leave the existing options as they were)